src/types/Post.ts

```diff
@@ -0,0 +1,11 @@
+import { Comment } from './Comment';
+import { User } from './User';
+
+export interface Post {
+  userId: number;
+  id: number;
+  title: string;
+  body: string;
+  user: User | null,
```

Comment:
```suggestion
  user?: User;
```

Reply:
@iPanda177 no, this is right types, `?` symbol said that value can be undefined, but we clearly save null here to define that object is not found